bin/pygrb/pycbc_grb_trig_cluster

```diff
@@ -168,105 +168,143 @@ outfile = os.path.join(
     ),
 )
 
-# -- generate clustering bins -------------------
+# this list contains the indexing of clusters from all slides
+all_clusters = []
 
-nbins = int((end - start) // win + 1)
-bins = [[] for i in range(nbins)]
-loudsnr = numpy.zeros(nbins)
-loudtime = numpy.zeros(nbins)
-clusters = []
+# load necessary information from all triggers
 
-# -- cluster ------------------------------------
-
-with HFile(args.trig_file, "r") as h5f:
-    time = h5f["network"]["end_time_gc"][()]
-    snr = h5f["network"][args.rank_column][()]
+with h5py.File(args.trig_file, "r") as h5f:
+    all_times = h5f["network"]["end_time_gc"][()]
+    all_snrs = h5f["network"][args.rank_column][()]
```

Comment:
In case you agree, here an f-string would do the job, h5f[f"network/{args.rank_column"}][()]

Reply:
`h5f[f"network/{args.rank_column}"][()]`